rhtconsulting/rhc-ose
# Auto-generate OSE inventory file

#### What does this PR do?

This role is used to generate a valid inventory file for an Openshift Enterprise install
#### How should this be manually tested?

Simply use the inventory file in the `openshift_install/test` dir(change and add to it as you wish) and run it with the playbook. Then check the inventory file in the dir you ran it from for inventory_<env_id> for your ose installer inventory file.
[jkwiatko@fedora23 openshift_install]$ ansible-playbook -i test/ose-provision openshift_install.yaml 
#### Is there a relevant Issue open for this?

N/A
#### Who would you like to review this?

/cc @etsauer  @sabre1041
